### PR TITLE
Enable shadowrealm testing for wasm api

### PR DESCRIPTION
**Pull request by rwaldron**
Starting with tests that don't have additional dependencies.

### CODE REVIEW

File: wasm/webapi/rejected-arg.any.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// META: global=window,worker
+// META: global=window,worker,shadowrealm
 
 for (const method of ["compileStreaming", "instantiateStreaming"]) {
```
Comment (Ms2ger) [inline wasm/webapi/rejected-arg.any.js:3]: I don't think you should run the webapi tests; these build on `Response` from fetch which I don't think we get in SRs.

File: wasm/serialization/module/serialization-via-notifications-api.any.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// META: global=window,dedicatedworker,shadowrealm
```
Comment (Ms2ger) [inline wasm/serialization/module/serialization-via-notifications-api.any.js:1]: Notifications aren't exposed, are they?

**Review by Ms2ger — CHANGES_REQUESTED** [review]

**Comment by rwaldron** [comment]
@Ms2ger thanks for the review! Changes made as requested

**Review by Ms2ger — COMMENTED** [review]
Thanks!

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**